internal/ethapi/api.go

```diff
@@ -1722,6 +1722,11 @@ func marshalReceipt(receipt *types.Receipt, blockHash common.Hash, blockNumber u
 	if receipt.ContractAddress != (common.Address{}) {
 		fields["contractAddress"] = receipt.ContractAddress
 	}
+
+	if tx.Type() == types.CeloDynamicFeeTxV2Type {
+		fields["baseFee"] = (*hexutil.Big)(receipt.BaseFee)
```

Comment:
CeloDyamicFeeV2Txs ocurring before the L2 start won't have the BaseFee field on the receipt, so what happens here when `receipt.BaseFee` is nil?

Reply:
it just sets `null` to `baseFee`. But added codes to omit the field if the tx type is `CeloDynamicFeeTxV2` and the tx is for Celo1